subprojects/build-init/src/main/groovy/org/gradle/buildinit/plugins/internal/GradlePluginProjectInitDescriptor.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.buildinit.plugins.internal;
+
+import org.gradle.api.internal.file.FileResolver;
+import org.gradle.util.GUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.gradle.buildinit.plugins.internal.BuildInitTestFramework.SPOCK;
+
+public class GradlePluginProjectInitDescriptor extends LanguageLibraryProjectInitDescriptor {
+    private static final Logger LOGGER = LoggerFactory.getLogger(GradlePluginProjectInitDescriptor.class);
+
+    public GradlePluginProjectInitDescriptor(TemplateOperationFactory templateOperationFactory,
+                                            FileResolver fileResolver,
```

Comment:
Nitpick: I've never seen white-spacing that lined variables with the left parenthesis. Usually they align to beginning of first type declaration.